[OSDOCS-17003] IBM Z CQA work (#116100)
openshift/openshift-docs: installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc

toc::[]

[role="_abstract"]
You can install an {product-title} cluster on {ibm-z-name} and {ibm-linuxone-name} by using a variety of different installation methods.

> Each method has qualities that can make them more suitable for different use cases, such as installing a cluster in a disconnected environment or installing a cluster with minimal configuration and provisioning.

Author: To get below the 300 character limit:
```suggestion
Choose the method that best fits your environment, such as a disconnected deployment or a minimally configured installation.
```
